spaces/anjmol/skin.scss

```
@@ -0,0 +1,9 @@
+.name {
```

This should be changed to the space id
```suggestion
.anjmol {
```